cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -293,7 +293,7 @@ for ifs in "${sriov_pfs[@]}"; do
   ifs_name="${ifs%%/device/*}"
   ifs_name="${ifs_name##*/}"
 
-  if [ $(echo "${PF_BLACKLIST[@]}" | grep -q "${ifs_name}") ]; then
+  if [ $(echo "${PF_BLACKLIST[@]}" | grep "${ifs_name}") ]; then
```

Comment:
You can also fix it by dropping the brackets and `$()`
```
if  echo "${PF_BLACKLIST[@]}" | grep -qs "${ifs_name}" ; then
...
```

Reply:
Thanks, i know, i prefer the method suggested in the PR.